Feature qttest tom

dwcaress opened the pull request.

tomoreilly commented:
Thanks Dave. 
Note that the version that displays the bathymetric surface with reduced glint isn't yet checked in - that required a change to the distributed Qt code, I'll figure out how to more easily incorporate it into qttest. 

Tom 



From: "David Caress" <notifications@github.com> 
To: "dwcaress" <MB-System@noreply.github.com> 
Cc: "Subscribed" <subscribed@noreply.github.com> 
Sent: Tuesday, May 26, 2020 4:44:38 PM 
Subject: [dwcaress/MB-System] Feature qttest tom (#1101) 


You can view, comment on, or merge this pull request online at: 


[ https://github.com/dwcaress/MB-System/pull/1101 | https://github.com/dwcaress/MB-System/pull/1101 ] Commit Summary 


    * added comments 
    * implement colormap control from View->ColorMap, including Haxby, Sealevel-1 
    * disable programmatic colormap control 
    * display test surface and set selectedFile.text to TEST SURFACE when no file specified on command line 
    * config changes 
    * good balance of ambient and specular light strength 
    * removed include of declarativesurface_p.h 
    * determine min/max coordinates using double rather than float comparison 
    * sub-sample dataset if it exceeds max QVector size (2^32 bytes) 
    * added debug statements, removed dead code 
    * added m_dataArray member 
    * commented out unneeded INCLUDEPATH additions 
    * implemented right-mouse pan function 
    * removed mouse radio buttons, added checkbox 
File Changes 


    * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-03c9588240ae391b1640ba4ab4ce0160 | src/qttest/BackEnd.cpp ] (32) 
    * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-7b130aded7e0590dd3ed5c70f2f549d6 | src/qttest/TopographicSeries.cpp ] (80) 
    * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-a7732f9ddf553523a6259dbe9b6d54b4 | src/qttest/TopographicSeries.h ] (9) 
    * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-28afedc8c64a72fe851fe3d2c509d3e3 | src/qttest/main.qml ] (242) 
    * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-71b6d8ae361c4522807c604b61e4459c | src/qttest/mbgrdviz-2.pro ] (14) 
    * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-2e28d808212ca5b8f4399401177b331b | src/qttest/mbgrdviz-2.pro.user ] (287) 
Patch Links: 


    * [ https://github.com/dwcaress/MB-System/pull/1101.patch | https://github.com/dwcaress/MB-System/pull/1101.patch ] 
    * [ https://github.com/dwcaress/MB-System/pull/1101.diff | https://github.com/dwcaress/MB-System/pull/1101.diff ] 


— 
You are receiving this because you are subscribed to this thread. 
Reply to this email directly, [ https://github.com/dwcaress/MB-System/pull/1101 | view it on GitHub ] , or [ https://github.com/notifications/unsubscribe-auth/AAS327FBGOREU3ZJEWRV2PDRTRH6NANCNFSM4NLMQYLA | unsubscribe ] .

dwcaress commented:
Tom,

Ok, I’ll wait. I was going to close the pull request anyway because the conflicts are complicated and I don’t want to spend time on that now. I’m making a beta release and I just thought I’d see if a sync would be easy.

Dave


> On May 26, 2020, at 4:51 PM, tomoreilly <notifications@github.com> wrote:
> 
> 
> Thanks Dave. 
> Note that the version that displays the bathymetric surface with reduced glint isn't yet checked in - that required a change to the distributed Qt code, I'll figure out how to more easily incorporate it into qttest. 
> 
> Tom 
> 
> 
> 
> From: "David Caress" <notifications@github.com> 
> To: "dwcaress" <MB-System@noreply.github.com> 
> Cc: "Subscribed" <subscribed@noreply.github.com> 
> Sent: Tuesday, May 26, 2020 4:44:38 PM 
> Subject: [dwcaress/MB-System] Feature qttest tom (#1101) 
> 
> 
> You can view, comment on, or merge this pull request online at: 
> 
> 
> [ https://github.com/dwcaress/MB-System/pull/1101 | https://github.com/dwcaress/MB-System/pull/1101 ] Commit Summary 
> 
> 
> * added comments 
> * implement colormap control from View->ColorMap, including Haxby, Sealevel-1 
> * disable programmatic colormap control 
> * display test surface and set selectedFile.text to TEST SURFACE when no file specified on command line 
> * config changes 
> * good balance of ambient and specular light strength 
> * removed include of declarativesurface_p.h 
> * determine min/max coordinates using double rather than float comparison 
> * sub-sample dataset if it exceeds max QVector size (2^32 bytes) 
> * added debug statements, removed dead code 
> * added m_dataArray member 
> * commented out unneeded INCLUDEPATH additions 
> * implemented right-mouse pan function 
> * removed mouse radio buttons, added checkbox 
> File Changes 
> 
> 
> * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-03c9588240ae391b1640ba4ab4ce0160 | src/qttest/BackEnd.cpp ] (32) 
> * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-7b130aded7e0590dd3ed5c70f2f549d6 | src/qttest/TopographicSeries.cpp ] (80) 
> * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-a7732f9ddf553523a6259dbe9b6d54b4 | src/qttest/TopographicSeries.h ] (9) 
> * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-28afedc8c64a72fe851fe3d2c509d3e3 | src/qttest/main.qml ] (242) 
> * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-71b6d8ae361c4522807c604b61e4459c | src/qttest/mbgrdviz-2.pro ] (14) 
> * M [ https://github.com/dwcaress/MB-System/pull/1101/files#diff-2e28d808212ca5b8f4399401177b331b | src/qttest/mbgrdviz-2.pro.user ] (287) 
> Patch Links: 
> 
> 
> * [ https://github.com/dwcaress/MB-System/pull/1101.patch | https://github.com/dwcaress/MB-System/pull/1101.patch ] 
> * [ https://github.com/dwcaress/MB-System/pull/1101.diff | https://github.com/dwcaress/MB-System/pull/1101.diff ] 
> 
> 
> — 
> You are receiving this because you are subscribed to this thread. 
> Reply to this email directly, [ https://github.com/dwcaress/MB-System/pull/1101 | view it on GitHub ] , or [ https://github.com/notifications/unsubscribe-auth/AAS327FBGOREU3ZJEWRV2PDRTRH6NANCNFSM4NLMQYLA | unsubscribe ] . 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> 

----------------------------------------------------
David W. Caress
Principal Engineer
Seafloor Mapping Lab

Monterey Bay Aquarium Research Institute
7700 Sandholdt Road
Moss Landing, CA 95039

caress@mbari.org
http://www.mbari.org/~caress/

Phone: 831-775-1775
